vocabularies/UI.xml

```diff
@@ -1832,6 +1832,56 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         </Property>
       </ComplexType>
 
+      <Term Name="Recommendations" Type="Edm.ComplexType" Nullable="false">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.Description" String="AI-based recommendations for an entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>This complex-typed annotation contains structural properties corresponding via name equality
+to non-key structural properties of the entity type for which recommendations are available.
+The type of such a property is a collection of a specialization of [`PropertyRecommendationType`](#PropertyRecommendationType).
```

Comment:
"logical specialization" as we don't support true specialization via `BaseType`?

Reply:
Instead of "logical specialization", I prefer the term "informal specialization". The opposite of logical is illogical.